ModelDesignWizard_python_QTableView/ModelDesignWizard.py

```diff
@@ -0,0 +1,590 @@
+# Copyright (C) 2022 The Qt Company Ltd.
+# SPDX-License-Identifier: LicenseRef-Qt-Commercial OR BSD-3-Clause
+
+import csv
```

Comment:
Woah, this is cool.  What would this require to run?  Could it be run with the openstudio cli or require a different python interpreter?   Would it load the Qt dlls we ship with the app?  I definitely think a python or ruby wizard UI would be preferable to a Qt one for wizards

Reply:
I think you can mix c++ and python qt elements. You're supposed to embbed a python interpreter in your c++ app I believe

Reply:
Here I was just trying to prototype faster (and try pyside6 which I hadn't yet).
But I see your point about being able to add your own wizards as an advanced user... interesting!

Reply:
I guess I'll try to call the openstudio CLI on it, provided a custom --python_path arg where I pip installed pyside6 and all.

Reply:
Ok cool, I was just trying to see where you were going with this.  QML/QtQuick might also be interesting to try at some point, it would let the UI be packaged with the measure and loaded at run time instead of compiled in the app.  Not trying to stop what you're doing, just throwing out another idea.

Reply:
This seems to work (locale aside)

```
$os_build_rel/Products/openstudio labs --python_path /home/julien/Virtualenvs/qt/lib/python3.8/site-packages ModelDesignWizard.py
```


Not sure that's really what we would want though, because it poses the problem of passing info from C++ to python and back... In this specific case, it's kinda fine because we could pass the link to the temp dir to python, and python could prepare the workflow.json (and call the CLI on it or not (C++ would do it)), but otherwise we'd always have to serialize the model (save) and reload when switching from C++ to python